### PR TITLE
move out all extensions declaration to its own config file

### PR DESCRIPTION
**Pull request by erwinmombay**
the shared configuration will be used by the old compilation tasks as well as the new single pass task

### CODE REVIEW

File: bundles.config.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+'use strict';
+/* global exports */
+
+exports.config = [
+  {'name': 'amp-3d-gltf', 'version': '0.1'},
+  {'name': 'amp-3q-player', 'version': '0.1'},
+  {'name': 'amp-access', 'version': '0.1', 'options': {'hasCss': true}},
+  {
+    'name': 'amp-access-laterpay',
+    'version': '0.1',
```
Comment (choumx) [inline bundles.config.js]: Nit: These two versions can be combined:

`'version': ['0.1', '0.2']`

Comment (erwinmombay) [reply bundles.config.js]: done.

**Comment by erwinmombay** [comment]
updated development documentation

**Comment by rsimha** [comment]
If presubmit complains about the console logs, maybe you could add the new file to the whitelist since it's not runtime code?

**Pull request merged**